docs/blog/posts/stable-release-announcement.md

```diff
@@ -0,0 +1,73 @@
+---
+date: 2024-03-10
+authors: [agardnerIT]
+description: >
+  In this blog post you will learn how to configure Keptn via the new Helm umbrella chart.
+comments: true
+---
+
+# Announcing the Official Release Candidate for Keptn v2
+
+The Keptn project is proud to announce a release candidate for what will become Keptn v2.
+We are sharing this release candidate
+so that the community and end users can participate in the testing
+to ensure that the actual release is as robust as possible.
+We anticipate (as long as no release blockers are identified)
+that Keptn v2 will be released approximately 1 month after this announcement.
```

Comment:
do we really want to make promises?

Reply:
yes, that's the ETA of the Stable Release

Reply:
this reads more like an estimation rather than a promise

Reply:
Yes, that also concerns me.  On the other hand, we kind of need to tell people how long they have to test this out and have their comments matter.